tests/parser/features/test_external_contract_calls.py

```diff
@@ -199,6 +199,28 @@ def __init__(arg1: address):
     print('Successfully executed a multiple external contract calls')
 
 
+def test_invalid_external_contract_call_to_the_same_contract(assert_tx_failed):
+    contract = """
+class Bar():
+    def bar() -> num: pass
+
+def bar() -> num:
+    return 1
+
+def _expr(x: address) -> num:
```

Comment:
What's the difference between `_stmt` and `_expr`?

Reply:
I fixed this.